.pre-commit-config.yaml

```diff
@@ -0,0 +1,104 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      - id: check-ast
+      - id: check-toml
+      - id: check-yaml
+        args:
+          - --allow-multiple-documents
+        exclude: "data/data_format_geo-entry.yaml"
+      - id: check-json
+      - id: check-executables-have-shebangs
+      - id: check-shebang-scripts-are-executable
+      - id: check-merge-conflict
```

Comment:
Do we need this, or doesn't GitHub do this already?

Reply:
I dont know if they do. I also could not find a documentation on this..
We can remove this, if you'd like.. 

This should never trigger, as with modern IDEs merging stuff is not a huge issue anymore imo..

This is something that is quite obvious in PRs and I assume, that neither you nor me will forget this, while merging a PR..
So this may be essentially wasted CPU-time..